entry.tsx

```diff
@@ -28,7 +28,10 @@ declare global {
 // This runs before we start to render anything.
 function preRenderSetup(callwhendone: () => void) {
     window.onerror = (msg, url, line, column, error) => {
-        if (msg === 'ResizeObserver loop limit exceeded') {
+        if (
+            msg === 'ResizeObserver loop limit exceeded' ||
+            msg === 'ResizeObserver loop completed with undelivered notifications.'
```

Comment:
What is this being added for? This error message seems to indicate an infinite loop which seems really concerning

Reply:
@hmhealey the issue was happening when we select "urgent" in the picker, which adds a new toogle in the ui. Floating-ui has to watch for element resizes to recompute the position.

According to this issue https://github.com/floating-ui/floating-ui/issues/1740#issuecomment-1151773854, it was safe to ignore.

I had tried the proposed solution `witdh: max-content` on the floating element, but that didn't solve the issue.

However, after merging latest master I can no longer reproduce :/ even with max-content removed :/ :/ ...
I'm removing the `msg === 'ResizeObserver loop completed with undelivered notifications.'`.

NOTE that the first condition to ignore was there beforehand, I have no idea why though.